dashboard/app/controllers/teacher_feedbacks_controller.rb

```diff
@@ -6,14 +6,23 @@ class TeacherFeedbacksController < ApplicationController
   # Feedback from any verified teacher who has provided feedback to the current
   # student on any level
   def index
-    @feedbacks_as_student = @teacher_feedbacks.order(created_at: :desc).includes(script_level: {lesson: :script}).select do |feedback|
+    scope = {
```

Comment:
@davidsbailey Can you point me to a good place to read about this?

Reply:
the effect of passing this to `includes` can be read about briefly here: https://api.rubyonrails.org/v6.1.1/classes/ActiveRecord/QueryMethods.html#method-i-includes the short version is that this syntax tells it which associated models to load, so that (in this case) when I call feedback.level.script_levels, no additional DB queries are needed. (the other nested fields are used by `summary_for_feedback`)

There are a few blog posts that go into way more detail about all the stuff that rails is doing behind the scenes when you use includes or other similar methods: https://bigbinary.com/blog/preload-vs-eager-load-vs-joins-vs-includes